Playbooks/AS-Incident-IP-Matched-on-Watchlist/azuredeploy.json

```diff
@@ -0,0 +1,327 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "AS-Incident-IP-Matched-on-Watchlist", 
+        "description": "This playbook is intended to be run from a Microsoft Sentinel incident. If any IP entities are found to match those in a specified watchlist containing a list of subnets, a comment noting this match will be added to the incident and the incident will be tagged with a label of your choosing.",
+        "prerequisites": "1. A Microsoft Sentinel watchlist containing subnets is needed",

```

Comment:
You can add post deployment step here as well,
"postDeployment":  ["Assign Microsoft Sentinel Contributor role to playbook over Log Analytics Workspace."]